packages/client-utils/test/sync-tools.test.js

```diff
@@ -13,13 +14,23 @@ import {
 const retryIntervalMs = 10;
 const DEFAULT_TIMEOUT = 30;
 
-const makeFakeFollow = () => {
-  let value = [[]];
+const makeFakeVstorageKit = () => {
+  let value;
 
   const setValue = newValue => (value = newValue);
+  // XXX remove this when we switch all sync-tools to use client-utils's vstorageKit
```

Comment:
[see the wiki](https://github.com/Agoric/agoric-sdk/wiki/Coding-Style#code-comments).  As an explicit statement about something to change shortly, this makes more sense as a TODO than as tech debt. OTOH, it doesn't rises to the level that deserves an issue. Tech debt (XXX) is usually for things that we might not ever get around to fixing.

```suggestion
  // TODO remove this when we switch all sync-tools to use client-utils's vstorageKit
```

Reply:
Resolved in https://github.com/Agoric/agoric-sdk/commit/228bf388dad60f5d798eb25fffffb29e04867a83